zerver/tests/test_muting_topics.py

```diff
@@ -12,10 +12,10 @@
     remove_topic_mute,
     topic_is_muted,
 )
-from zerver.models import MutedTopic, UserProfile, get_stream
+from zerver.models import UserProfile, UserTopic, get_stream
 
 
-class MutedTopicsTests(ZulipTestCase):
+class UserTopicsTests(ZulipTestCase):
```

Comment:
The test class might want to stay the same here.

Reply:
Yep, fixed